frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewMetaData/TagRow.tsx

```diff
@@ -99,80 +96,77 @@ export const TagRow = ({ feature }: IFeatureOverviewSidePanelTagsProps) => {
         }
     };
 
+    const addTagButton = (
+        <StyledAddTagButton
+            size='small'
+            permission={UPDATE_FEATURE}
+            projectId={feature.project}
+            variant='text'
+            onClick={() => {
+                setManageTagsOpen(true);
+            }}
+        >
+            <StyledAddIcon /> Add tag
+        </StyledAddTagButton>
+    );
+
     return (
         <>
-            <ConditionallyRender
-                condition={!tags.length}
-                show={
-                    <StyledMetaDataItem>
-                        <StyledMetaDataItemLabel>Tags:</StyledMetaDataItemLabel>
-                        <StyledPermissionButton
-                            size='small'
-                            permission={UPDATE_FEATURE}
-                            projectId={feature.project}
-                            variant='text'
-                            onClick={() => {
-                                setManageTagsOpen(true);
-                            }}
-                        >
-                            Add tag
-                        </StyledPermissionButton>
-                    </StyledMetaDataItem>
-                }
-                elseShow={
-                    <StyledTagRow>
-                        <StyledMetaDataItemLabel>Tags:</StyledMetaDataItemLabel>
-                        <StyledTagContainer>
-                            {tags.map((tag) => {
-                                const tagLabel = `${tag.type}:${tag.value}`;
-                                return (
-                                    <Tooltip
-                                        key={tagLabel}
-                                        title={
-                                            tagLabel.length > 35 ? tagLabel : ''
-                                        }
-                                        arrow
-                                    >
-                                        <StyledAddedTag
-                                            label={tagLabel}
-                                            size='small'
-                                            deleteIcon={
-                                                <Tooltip
-                                                    title='Remove tag'
-                                                    arrow
-                                                >
-                                                    <ClearIcon />
-                                                </Tooltip>
+            {!tags.length ? (
+                <StyledMetaDataItem>
+                    <StyledLabel>Tags:</StyledLabel>
+                    <StyledTagContainer>{addTagButton}</StyledTagContainer>
+                </StyledMetaDataItem>
+            ) : (
+                <StyledTagRow>
+                    <StyledLabel>Tags:</StyledLabel>
```

Comment:
Small point, but is there a reason why you swap out the metadata item container here? 

In fact, I'd expect these two branches to be exactly the same: Container -> Label + Tag Container -> Tags + Button. So if the list of tags is empty, then `tags.map` gives you no components back and it'll still work.

Does that check out?

Reply:
It didn't align tags properly, because we have 2 cases here: if tags fit in one line, keep it this way (aligned to the right), but if it doesn't fit - align to the left in starting from next line.
I'll spend 5min checking if I can make it work.